service/controller/aws/v11/resource/chartconfig/current.go

```diff
@@ -19,6 +20,14 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
+	if key.IsDeleted(customObject.ObjectMeta) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", "redirecting chartconfig deletion to provider operators")
+		resourcecanceledcontext.SetCanceled(ctx)
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+
+		return nil, nil
+	}
```

Comment:
chartconfig and configmap resources are simpler. So we just cancel the resource in GetCurrentState.